test/types/mongodb.test-d.ts

```diff
@@ -19,9 +20,14 @@ expectDeprecated(Db.prototype.unref);
 expectDeprecated(MongoDBDriver.ObjectID);
 expectNotDeprecated(MongoDBDriver.ObjectId);
 
+interface TSchema extends Document {
+  name?: string;
```

Comment:
I think this needs to have at least one non-optional parameter, because otherwise a generic document type might still match the TSchema

Reply:
(This is more of an issue for something like expectAssignable and isn't a concern here where we use expectType, since the type check is strict, but better safe than sorry)

Reply:
Updated to make it required.